src/main/java/uk/gov/hmcts/reform/hmc/config/MessageReaderFromTopicConfiguration.java

```diff
@@ -0,0 +1,93 @@
+package uk.gov.hmcts.reform.hmc.config;
+
+import com.azure.messaging.servicebus.ServiceBusClientBuilder;
+import com.azure.messaging.servicebus.ServiceBusErrorContext;
+import com.azure.messaging.servicebus.ServiceBusException;
+import com.azure.messaging.servicebus.ServiceBusFailureReason;
+import com.azure.messaging.servicebus.ServiceBusProcessorClient;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessage;
+import com.azure.messaging.servicebus.ServiceBusReceivedMessageContext;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.context.event.ApplicationStartedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.hmc.ApplicationParams;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Component
+public class MessageReaderFromTopicConfiguration {
+
+    private final ApplicationParams applicationParams;
+
+    public MessageReaderFromTopicConfiguration(ApplicationParams applicationParams) {
+        this.applicationParams = applicationParams;
+    }
+
+    @Async
+    @EventListener(ApplicationStartedEvent.class)
+    @SuppressWarnings("squid:S2189")
+    public void readMessageFromTopic() {
+        CountDownLatch countdownLatch = new CountDownLatch(1);
+
+        // Create an instance of the processor through the ServiceBusClientBuilder
+        ServiceBusProcessorClient processorClient = new ServiceBusClientBuilder()
+            .connectionString(applicationParams.getConnectionString())
+            .processor()
+            .topicName(applicationParams.getTopicName())
+            .subscriptionName(applicationParams.getSubscriptionName())
+            .processMessage(MessageReaderFromTopicConfiguration::processMessage)
+            .processError(context -> processError(context, countdownLatch))
+            .buildProcessorClient();
+
+        log.info("Starting the processor");
+        processorClient.start();
+
+        try {
+            TimeUnit.SECONDS.sleep(10);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+        log.info("Stopping and closing the processor");
+        processorClient.close();
```

Comment:
We don't want to be waiting and/or stopping the processing - as long as the CFT Hearing Service is running it should be accepting and processing messages